cmd/validator/core/command.go

```diff
@@ -45,7 +45,7 @@ func init() {
 	ValidatorCmd.Flags().StringVar(&addr, "addr", "", "If provided validate a single address instead of reading Stdin.")
```

Comment:
A resource override might be nice too, so that the shell script to process/pipe an error log back into the validator doesn't need to remember the format.

I wouldn't consider either of these required for this PR, someone can improve the tool later if/when it becomes a problem.

Reply:
I put the box command in and will leave the resource override for another conversation.